GitUI/CommandsDialogs/BrowseDialog/GitStatusMonitor.cs

```diff
@@ -263,6 +283,7 @@ void commandsSource_GitUICommandsChanged(object sender, GitUICommandsChangedEven
                     oldCommands.PreCheckoutRevision -= GitUICommands_PreCheckout;
                     oldCommands.PostCheckoutBranch -= GitUICommands_PostCheckout;
                     oldCommands.PostCheckoutRevision -= GitUICommands_PostCheckout;
+                    oldCommands.PostRepositoryChanged -= GitUICommands_PreRepositoryChanged;
```

Comment:
I have not digged into the code. But where is `+= GitUICommands_PreRepositoryChanged`?

Reply:
GitUICommands_PreRepositoryChanged is never executed because there is no PreRepositoryChanged event...
Fixed the handling